Assets/MRTK/SDK/Experimental/Elastic/Scripts/ElasticsManager.cs

```diff
@@ -134,28 +134,30 @@ public bool EnableElasticsUpdate
         /// Applies elastics calculation to the passed targetTransform and applies to the host transform.
         /// </summary>
         /// <param name="targetTransform">Precalculated target transform that's influenced by elastics</param>
+        /// <param name="filter">If a filter is passed only the given transform type is going to be calculated.</param>
         /// <returns>Modified transform modes.</returns>
-        public TransformFlags ApplyTargetTransform(MixedRealityTransform targetTransform)
+        public TransformFlags ApplyTargetTransform(MixedRealityTransform targetTransform, TransformFlags filter = TransformFlags.Move|TransformFlags.Rotate|TransformFlags.Scale)
         {
             Debug.Assert(hostTransform != null, "Can't apply target before calling Initialize with a valid transform reference.");
             if (hostTransform != null)
             {
-                if (elasticTypes.HasFlag(TransformFlags.Move))
+                TransformFlags enabledFlags = filter & elasticTypes;
+                if (enabledFlags.HasFlag(TransformFlags.Move))
```

Comment:
Nit: Maybe consider better nomenclature for fields of type TransformFlags? Here they are being called filter, type (elasticTypes) and flag (enabledFlags) ... kind of suggesting they are variables of different nature?

Reply:
renamed